modules/statics/src/coins.ts

```diff
@@ -15152,6 +15152,24 @@ export const coins = CoinMap.fromCoins([
     UnderlyingAsset['sol:ks'],
     SOL_TOKEN_FEATURES
   ),
+  solToken(
+    '92bfdced-9b06-47ce-968c-a41e83f787df',
+    'sol:apusdt',
+    'Wrapped USDT (Allbridge from Polygon)',
```

Comment:
```suggestion
    'Bridged Tether (Allbridge)',
```
Based on the solana explorer name https://explorer.solana.com/address/DNhZkUaxHXYvpxZ7LNnHtss8sQgdAfd1ZYS1fB7LKWUZ, but its up to Daniel to confirm the name i guess

Reply:
I think it depends of the explorer? Daniel shared with solscan where have Wrapped USDT name: https://solscan.io/token/DNhZkUaxHXYvpxZ7LNnHtss8sQgdAfd1ZYS1fB7LKWUZ